__tests__/app.test.js

```diff
@@ -342,4 +342,116 @@ describe('7. GET /api/users', () => {
         });
       });
   });
+});
+
+
+describe('8. GET /api/articles?topic=input', () => {
+  test('status:200, responds with all articles under a certain topic', () => {
+    return request(app)
+      .get('/api/articles?topic=mitch')
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
```

Comment:
Please make sure you assert against the length of the articles array, otherwise you are at risk of this passing as a false positive. Either checking the exact length or making sure it's at least greater than zero - without such a check, these tests are not sufficient.

**NOTE:** this applies to lots of the other tests also!